libcontainerd/container_windows.go

```diff
@@ -41,6 +41,14 @@ func (ctr *container) start() error {
 		return err
 	}
 
+	if ctr.ociSpec.Windows.Servicing {
+		if err := hcsshim.ShutdownComputeSystem(ctr.containerID, hcsshim.TimeoutInfinite, ""); err != nil {
```

Comment:
Can you add a comment next to L37 that says `hcsshim.StartComputeSystem` acts differently when passed the servicing indicator so that it blocks indefinitely. As opposed to a "normal" container start which does not.